client/my-sites/marketing/index.js

```diff
@@ -33,20 +31,14 @@ export default function() {
 		'/sharing/buttons',
 	];
 
-	if ( config.isEnabled( 'marketing/tools' ) ) {
-		paths.push( '/marketing/tools' );
-	}
+	paths.push( '/marketing/tools' );
```

Comment:
We no longer need to keep that line, we should just add this path when `paths` is declared above.